From 8f2a5c14abadc4825a0c836c5b8c0600fa04fd94 Mon Sep 17 00:00:00 2001
From: Chris Burr <christopher.burr@cern.ch>
Date: Mon, 28 Feb 2022 15:53:49 +0100
Subject: [PATCH 2/2] Fix installing shared library symlinks with DESTDIR

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 599b145..5a39e49 100644
--- a/Makefile
+++ b/Makefile
@@ -200,7 +200,7 @@ ifeq ($(NO_SOLIB),0)
 	$(INSTALL) -D $(LIBNAME) $(DESTDIR)$(LIBDIR)/$(LIBNAME)
 	$(LNS) $(LIBNAME) $(DESTDIR)$(LIBDIR)/$(SONAME)
 	mkdir -p $(DESTDIR)$(USRLIBDIR)
-	$(LNS) $(LIBDIR)/$(SONAME) $(DESTDIR)$(USRLIBDIR)/$(DEVELLIB)
+	$(LNS) $(SONAME) $(DESTDIR)$(USRLIBDIR)/$(DEVELLIB)
 	sed \
 	-e 's,@VERSION\@,$(VERSION),g' \
 	-e 's,@prefix\@,$(PREFIX),g' \
